README.md

```diff
@@ -59,8 +59,8 @@ ansible-playbook -i inventory/hosts playbooks/upgrade.yml
 # Further reading
 
 ## Complete Production Installation Documentation:
-- [OpenShift Container Platform](https://docs.openshift.com)
-- [OpenShift Origin](https://docs.okd.io)
+- [OpenShift Container Platform](https://docs.openshift.com/container-platform/latest/install/index.html)
+- [OKD](https://docs.okd.io/latest/install/index.html) (formerly OpenShift Origin)
```

Comment:
OKD and Origin are both active names

Reply:
Please let's have a short exchange [here](https://github.com/openshift/openshift-ansible/pull/11513/files/d986aff2787aca6e1e43927f51c0cda231bb300f), and I'll adjust the PRs accordingly.